srcpkgs/emilua/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'emilua'
+pkgname=emilua
+version=0.4.3
+revision=1
+archs="x86_64"
```

Comment:
This should not be limited to one architecture and libc unless there is a hard limitation.

Reply:
Fixed it, only build for x86_64-glibc and aarch64 at the moment.